middleware/helpers.js

```diff
@@ -80,7 +80,7 @@ module.exports.getParameterValue = function (version, parameter, pathKeys, match
   case 'form':
   case 'formData':
     if (paramType.toLowerCase() === 'file') {
-      val = req.files[parameter.name];
+      val = req.files[parameter.name] && req.files[parameter.name].length ? req.files[parameter.name][0] : undefined; // File data is missing
```

Comment:
I don't think this is necessary.  You should be able to just leave it as-is.  If `req.files[parameter.name]` can be `undefined`?  You could still simplify this to `req.files[parameter.name] ? req.files[parameter.name][0] : undefined`;  Minor nit but in case you can benefit from it, I figured I'd share.